modules/RateLimiter/config.py

```diff
@@ -0,0 +1,10 @@
+CONNECTION = {
+    "host_address": "127.0.0.1", # Redis DB IPv4 address
+    "port": 1234, # Redis DB port
+    "db": 0 # Redis DB ID
+}
+LIMITING = {
+    "key_prefix": "ratelimit:", # Prefix for rate limiter keys
+    "max_reqs": 30, # Maximum number of requests in the expiry period
```

Comment:
How is this number decided?

Reply:
Arbitrary, will need to be decided at some point. I'll add a todo there and create a ticket to decide (probs through some mild use case investigation)

Reply:
Remember to chase Rong for the documentation regarding IP :p

Reply:
Yep, indeed. Closing the loop on determining an acceptable value, have created an investigation ticket here: <https://jkl-tl.atlassian.net/browse/REDIS-37>